### DISCUSSION TITLE
Bitbucket Cloud permissions in documentation inaccurate

### DISCUSSION THREAD

**Discussion by tommyallieri** [renovatebot/renovate#25055] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.5

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

We've been following the instructions for integrating the renovate bot with bitbucket cloud (https://docs.renovatebot.com/modules/platform/bitbucket/), however, group option doesn't work.

The "Create repositories" permission is getting deprecated soon and didn't provide the write access to the git repositories it required to manage branch/PRs on our behalf, instead throwing errors "ERROR: Repository is not found".

Upgrading the group to have write access to the necessary git repositories resolved the problem.

### Relevant debug logs

ERROR: Repository is not found

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
Source code for the docs is here: https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/bitbucket/index.md?plain=1

PRs welcome!
